src/main/features/response/views/check-and-send.njk

```diff
@@ -63,6 +63,38 @@
       {% endif %}
 
       {{ row(label = 'Why do you reject all of this claim?', value = rejectAllOption, changeLink = ResponsePaths.defenceRejectAllOfClaimPage.evaluateUri({ externalId: claim.externalId }), bold = true) }}
+
+      {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.ALREADY_PAID %}
+        {{
+          row(
+            label = 'How much have you paid?',
+            value = t(draft.rejectAllOfClaim.howMuchHaveYouPaid.amount | numeral),
```

Comment:
Amounts do not need to be passed through `t()`. There are same in English and Welsh

Reply:
Ahh, that's what `t()` does!